tests/00-setup

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x 
+
+# Check if amulet is installed before adding repository and updating apt-get.
+dpkg -s amulet
```

Comment:
Oh this is brilliant. Did you add this or did this come stock with the new templates?